views/js/pciCreator/ims/audioRecordingInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -90,7 +105,7 @@
 
 <div class="panel">
     <label for="max-records" class="spinner">{{__ "Max attempts:"}}</label>
-    <input id="max-records" name="maxRecords" value="{{maxRecords}}" class="large" data-increment="1" data-min="0" type="text" />
+    <input id="max-records" name="maxRecords" value="{{maxRecords}}" class="large" data-increment="1" data-min="0" type="text" disabled />
```

Comment:
Should this `disabled` be here? Or only in a certain condition?

Reply:
Yes, it will be enabled in `Question.js --> `initForm -->`$form.find('input[name="maxRecords"]').prop('disabled', false);`
If intially enabled, it was flickering a little (not that it's important).
Can remove if you think it's better to.

Reply:
Ok to leave it, as long as it's intended.